routes/index-routes.js

```diff
@@ -1,35 +0,0 @@
-const { compareSync } = require("bcrypt");
-const express = require("express");
-const Company = require("../models/company");
-const router = express.Router();
-const { averagesObject, getOverallReviewScore } = require("../public/javascripts/averages");
-
-// GET route for rendering the homepage. We get the companies from the DB and run the functions to calculate the averages.
-
-router.get("/", (req, res, next) => {
-  Company.find()
-    .populate("reviews")
-    .lean()
-    .then((companiesFromDb) => {
-      companiesFromDb.forEach((company, index) => {
-        if (company.reviews) {
-          company.averages = averagesObject(company.reviews);
-          company.overall = getOverallReviewScore(company.reviews);
-        }
-        company.index = index + 1;
```

Comment:
This seems fishy. You get `index` as default from your `forEach` loop. I can't really understand why you're changing here but I'd really advise not to, seems hacky...